### PR TITLE
Use nedb and connect

### PR DESCRIPTION
**Pull request by ketanbhatt**
https://github.com/ketanbhatt/habits/issues/8

### CODE REVIEW

File: constants.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 module.exports = {
   userNameKey: 'userName',
+  habitCollection: 'habits',
```
Comment (WhiteHatArpit) [inline constants.js]: Where will we use this?

File: main.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const appRoot = require('app-root-path');
+
```
Comment (WhiteHatArpit) [inline main.js:5]: This should be used for line 11 & 12.

File: package.json [hunk 3]
```diff
@@ -13,6 +13,8 @@
     "eslint-plugin-react": "^4.1.0"
   },
   "dependencies": {
-    "nconf": "^0.8.4"
+    "app-root-path": "^1.0.0",
```
Comment (WhiteHatArpit) [inline package.json:16]: Do we need to use this?
Its not being used with consistency for now.

File: constants.js (hunk 1 above)
Comment (ketanbhatt) [inline constants.js]: My bad. Was using with Loki. Will remove

File: package.json (hunk 3 above)
Comment (ketanbhatt) [inline package.json:16]: We are using it for constants, configuration, path for database. I think it is great! Earlier had to to `../../constants.js`

Comment (WhiteHatArpit) [inline package.json:16]: Cool!

File: .gitignore [hunk 4]
```diff
@@ -34,3 +34,5 @@ node_modules
 
 .idea/
 .DS_Store
+
+habits-config.json
```
Comment (WhiteHatArpit) [inline .gitignore]: I think this config file should be inside the database directory too.

File: main.js (hunk 2 above)
Comment (ketanbhatt) [inline main.js:5]: pulled your changes from master, that fucked it.

Comment (ketanbhatt) [inline main.js:5]: Now will have to check everywhere :x:

File: .gitignore (hunk 4 above)
Comment (ketanbhatt) [inline .gitignore]: Can be. On second thoughts, even the database should be out of git too. We will configure all of that towards the end. What say?

**Comment by ketanbhatt** [comment]
@TigerApps Removed config and db files from git. Otherwise everytime we test something it will change db and thus be counted as a change. 

So cool :dancer:

**Pull request merged**